tools/doc/generate.js

```diff
@@ -49,9 +49,15 @@ args.forEach(function(arg) {
   } else if (arg.startsWith('--output-directory=')) {
     outputDir = arg.replace(/^--output-directory=/, '');
   } else if (arg.startsWith('--apilinks=')) {
-    apilinks = JSON.parse(
-      fs.readFileSync(arg.replace(/^--apilinks=/, ''), 'utf8')
-    );
+    const filename = arg.replace(/^--apilinks=/, '');
+    try {
+      apilinks = JSON.parse(
+        fs.readFileSync(filename, 'utf8')
+      );
+    } catch (e) {
+      console.log(`Failure reading ${filename}, maybe remove it?`);
```

Comment:
I don't think removing it always solve the issue, depending on how the file is generated the next time. Usually when I run into this it's because the file is empty because the content is piped from stdout instead of being written by the bogus binary (in which case it would fail properly instead of piping out an empty file).

Alternative solution: https://github.com/nodejs/node/pull/25019